internal/auditd/sessiontracker/sessiontracker.go

```diff
@@ -324,11 +327,12 @@ func (o *sessionTracker) DeleteRemoteUserLoginsBefore(t time.Time) {
 }
 
 type user struct {
-	added  time.Time
-	srcPID int
-	hasRUL bool
-	login  common.RemoteUserLogin
-	cached []*aucoalesce.Event
+	added    time.Time
+	srcPID   int
+	hasRUL   bool
+	login    common.RemoteUserLogin
+	cached   []*aucoalesce.Event
+	procArgs []string
```

Comment:
You can remove this field - there is no need to store process arguments in this struct.

Reply:
done